app/views/job_templates/_job_template_callback_tab_headers.html.erb

```diff
@@ -0,0 +1 @@
+<li><a href="#ansible_callback_enabled" data-toggle="tab"><%= _("Ansible Callback") %></a></li>
```

Comment:
The name predestines the entire tab to never having anything else in it apart from the single checkbox. How about we go with more generic "Ansible"?